packages/compass-aggregations/src/components/aggregation-side-panel/stage-wizard-use-cases/sort/sort.tsx

```diff
@@ -117,7 +117,7 @@ export const SortForm = ({
   const comboboxStyles = useMemo(() => {
     return {
       width: `calc(${String(
-        Math.max(...fields.map((label) => label.length))
+        Math.max(...fields.map((label) => label.length), 10)
```

Comment:
This is to make sure that we have a minimum desirable width for the select fields when all the field names are 1/2 letters only.